force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -161,18 +210,19 @@ public with sharing class FlowControlLoopsCollections {
      * @return The new string with the inputString repeated repeatCount times.
      */
     public static String repeatString(String inputString, Integer repeatCount) {
-        // Initialize the result String
-
-        // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
-            // If it is not the last iteration, add a semicolon to separate the strings
+       String resultString = '';
+        
+        for (Integer i=0; i< repeatCount; i++) {
+
+           if (i == repeatCount - 1) {                 //final pass, omit "; "
+            resultString = resultString + inputString;
+           } else {
+            resultString = resultString + inputString +'; ';
```

Comment:
You can also check only for the condition that you want to add part that is different in your check. That way you can always append the inputString, and only append the ; if you need to. Prevents duplicating the inputString portion again. This is more useful when the logic might be more complicated, but it is good to get into that mindset of what is the same and what is different.